pulpcore/pulpcore/app/viewsets/base.py

```diff
@@ -169,7 +169,7 @@ def urlpattern(cls):
     def endpoint_pieces(cls):
         # This is a core ViewSet, not Master/Detail. We can use the endpoint as is.
         if cls.queryset.model._meta.master_model is None:
-            return (cls.endpoint_name,)
+            return [cls.endpoint_name]
```

Comment:
Function should preferably return only one type (could return either tuple or list).  Since the length of the return value is variable, we should probably use a list (it doesn't really matter but it's more intuitive).